kas.yml

```diff
@@ -4,7 +4,7 @@ header:
 distro: pce-base
 machine: pcengines-apu2
 target:
-  - core-image-minimal
+  - apu2-diagnostic-image
```

Comment:
@Asiderr Please create another kas file, as explained in the docker PR. Let me know if something is not clear.